var/spack/repos/builtin/packages/libxsmm/package.py

```diff
@@ -82,7 +82,14 @@ def build(self, spec, prefix):
         ]
 
         # JIT (AVX and later) makes MNK, M, N, or K spec. superfluous
-#       make_args += ['MNK=1 4 5 6 8 9 13 16 17 22 23 24 26 32']
+        # make_args += ['MNK=1 4 5 6 8 9 13 16 17 22 23 24 26 32']
+
+        if '%aocc' in spec:
+            make_args += ['GNU=1']
+            make_args += ['AVX=2']
```

Comment:
I think this option should be conditional on the target.

Reply:
Agree. Addressed the same.